src/openvic-simulation/politics/Rebel.cpp

```diff
@@ -1,5 +1,10 @@
 #include "Rebel.hpp"
 
+#include <string>
+
+#include "misc/Modifier.hpp"
+
+
```

Comment:
```suggestion
#include "openvic-simulation/misc/Modifier.hpp"

```
Can you add `openvic-simulation` to the include path, also `<string>` isn't needed because the `identifier` variable in `generate_modifiers` should be a `std::string_view`

Reply:
I think vs code created the import. Is there a setting to include openvic-simulation?

Reply:
I think it's `clangd` doing it, it sometimes does it for me as well despite the includes not being necessary. If you find a way to disable it please let me know